pkg/controllers/provisioning/v1alpha1/vars.go

```diff
@@ -0,0 +1,35 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"time"
+)
+
+const (
+	TimeFormat = time.RFC3339
+	TTLSeconds = 300
+)
+
+var (
+	IgnoredOwners []schema.GroupVersionKind = []schema.GroupVersionKind{
+		{Group: "apps", Version: "v1", Kind: "DaemonSet"},
+	}
+	ProvisionerNameLabelKey      = v1alpha1.SchemeGroupVersion.Group + "/name"
+	ProvisionerNamespaceLabelKey = v1alpha1.SchemeGroupVersion.Group + "/namespace"
+	ProvisionerTTLKey            = v1alpha1.SchemeGroupVersion.Group + "/ttl"
```

Comment:
This TTL label should probably live where it's used.

Reply:
got it.

Reply:
It's being used multiple places, so I'd like to keep it here.

Reply:
The TTL is not scoped to the provisioner. It's scoped to the deallocator.